damusTests/EventGroupViewTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EventGroupViewTests.swift
+//  damusTests
+//
+//  Created by Terry Yiu on 2/26/23.
+//
+
+import XCTest
+@testable import damus
+
+final class EventGroupViewTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testReactingToText() throws {
```

Comment:
Not exhaustive on the tests yet, but I wanted to push out this PR so that Will could pull it in. I tested loading the app in Arabic and pulled up the event grouped Notifications view without issue.